backend/src/environments/environment.dev.ts

```diff
@@ -12,7 +12,10 @@ export const environment : Environment =
 	apiRoutes:
 	{
 		todos: '/api/todos',
-		todosWithId: '/api/todos/:todoId'
+		todosWithId: '/api/todos/:todoId',
+		register: '/register',
+		login: '/login',
+		apiWithId: '/register/:userId'
```

Comment:
register: '/register' without userId ... please send erverything via POST

Reply:
I'm not clear on this one